changelogs/fragments/7012-sorcery-grimoire-mgmt.yml

```
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - "``community.general.sorcery`` - grimoire (repository) management support"
+bugfixes:
+  - "``community.general.sorcery`` - fix queue generation before the whole system rebuild"
```

Please note that it's better to move bugfixes to individual PRs. These can be backported to both stable-7 and stable-6, while features can only be backported to stable-7.
```suggestion
  - "sorcery - fix queue generation before the whole system rebuild (https://github.com/ansible-collections/community.general/pull/7012)."
```